tests/Unit/DataStructures/Test_ApplyMatrices.cpp

```diff
@@ -34,7 +34,7 @@ constexpr Spectral::Basis basis = Spectral::Basis::Legendre;
 // Using Gauss points for this test would be nice, since that would
 // let us test the extent=1 cases, but that needs to wait for
 // everything to be updated to use Mesh.
-constexpr Spectral::Quadrature quadrature = Spectral::Quadrature::GaussLobatto;
```

Comment:
Remove or revise the comment that says 'Using Gauss points for this test would be nice...'